### PR TITLE
[release] updates changelog for 0.2.118

### PR DESCRIPTION
**Pull request by iris-garden**

### CODE REVIEW

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.118
+
+Released 2023-05-30
```
Comment (jigold) [inline hail/python/hail/docs/change_log.md:38]: Can you change the date?

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -33,6 +33,18 @@ Please note that **forward compatibility should not be expected, especially
...
+
+### Bug Fixes
+
+- (hail#13113) `hail.ggplot2.geom_point` now displays a legend group for a column even when it has only one value in it.
+- (hail#13089) Requester pays buckets now work in `hailtop.fs` and `hl.hadoop_*`. This has been broken since at least 0.2.115.
+- (hail#12995) Fixed bug causing poor performance and memory leaks for `Matrix.annotate_rows` aggregations.
```
Comment (jigold) [inline hail/python/hail/docs/change_log.md:44]: What about #12569 as new features? 
#13081 as a new feature?

I don't think #13126 is a bug fix, but @chrisvittal can chime in?

**Review by jigold — CHANGES_REQUESTED** [review]

File: hail/python/hail/docs/change_log.md (hunk 2 above)
Comment (iris-garden) [reply hail/python/hail/docs/change_log.md:44]: #13113 was the reason I wanted to cut a release in the first place, so I'd like to wait for it to go in first.

Comment (iris-garden) [reply hail/python/hail/docs/change_log.md:45]: Added changelog messages for https://github.com/hail-is/hail/pull/12569 and https://github.com/hail-is/hail/pull/13126. I'll change the date as well once this is actually ready to go in. This PR is WIP pending the merge of at least https://github.com/hail-is/hail/pull/13113, https://github.com/hail-is/hail/pull/13147, and any other PRs that are necessary to make CI work again. We could also wait for https://github.com/hail-is/hail/pull/13089 and https://github.com/hail-is/hail/pull/12995, but I don't have strong opinions either way about that. @danking, thoughts?

File: hail/python/hail/docs/change_log.md [hunk 3]
```diff
@@ -33,6 +33,20 @@ Please note that **forward compatibility should not be expected, especially
...
+## Version 0.2.118
+
+Released 2023-05-30
+
+### Bug Fixes
+
+- (hail#13126) Query-on-Batch pipelines with one partition are now retried when they encounter transient errors.
+- (hail#13113) `hail.ggplot.geom_point` now displays a legend group for a column even when it has only one value in it.
+- (hail#13089) Requester pays buckets now work in `hailtop.fs` and `hl.hadoop_*`. This has been broken since at least 0.2.115.
+- (hail#12995) Fixed bug causing poor performance and memory leaks for `Matrix.annotate_rows` aggregations.
```
Comment (danking) [inline hail/python/hail/docs/change_log.md:45]:
```suggestion
```

File: hail/python/hail/docs/change_log.md [hunk 4]
```diff
@@ -33,6 +33,20 @@ Please note that **forward compatibility should not be expected, especially
...
 
+## Version 0.2.118
+
+Released 2023-05-30
+
+### Bug Fixes
+
+- (hail#13126) Query-on-Batch pipelines with one partition are now retried when they encounter transient errors.
+- (hail#13113) `hail.ggplot.geom_point` now displays a legend group for a column even when it has only one value in it.
+- (hail#13089) Requester pays buckets now work in `hailtop.fs` and `hl.hadoop_*`. This has been broken since at least 0.2.115.
```
Comment (danking) [inline hail/python/hail/docs/change_log.md:44]:
```suggestion
- (hail#13075) (hail#13074) Add a new transient error plaguing pipelines in Query-on-Batch in Google: `java.net.SocketTimeoutException: connect timed out`.
```

File: hail/python/hail/docs/change_log.md [hunk 5]
```diff
@@ -33,6 +33,20 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.118
+
+Released 2023-05-30
```
Comment (danking) [inline hail/python/hail/docs/change_log.md:38]:
```suggestion
Released 2023-05-30

### New Features

- (hail#13140) Enable `hail-az` and Azure Blob Storage `https` URLs to contain SAS tokens to enable bearer-auth style file access to Azure storage.
- (hail#13129) Allow subnet to be passed through to gcloud in hailctl.
```

**Comment by danking** [comment]
@iris-garden bump!

**Review by danking — APPROVED** [review]

**Pull request merged**